File: 14 - Tax Calculator/C/main.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    float amount, tax = 0;
    printf("What is the amount? $");
    scanf("%f", &amount);

    char state[2];
    printf("What is the state? ");
    scanf("%2s", state);

    if ( strcmp(state, "WI") == 0 ) {
        tax = amount * 0.055;
        printf("The tax is $%.2f.\n", tax);
    }

    printf("The total is $%.2f.\n", amount + tax);

    return 0;
}

File: 14 - Tax Calculator/Golang/main.go
package main

import (
    "fmt"
)

func main() {
    var amount, tax float64
    fmt.Printf("What is the amount? $")
    fmt.Scanf("%f", &amount)

    var state string
    fmt.Printf("What is the state? ")
    fmt.Scanf("%s", &state)
    if (state == "WI") {
        tax = 0.055 * amount;
        fmt.Printf("The tax is $%.2f\n", tax)
    }

    fmt.Printf("The total is $%.2f\n", amount + tax)
}
